packages/@o3r/core/builders/ngc/index.ts

```diff
@@ -1,8 +1,8 @@
 import { BuilderOutput, createBuilder } from '@angular-devkit/architect';
 import { main } from '@angular/compiler-cli/src/main';
+import * as ts from '@schematics/angular/third_party/github.com/Microsoft/TypeScript/lib/typescript';
```

Comment:
Should we use `ts` from `typescript` or from here ?

Reply:
I followed what was done previously but you are right, there is not good reason to do so